src/backend/mailbox_manager/management/commands/setup_dimail_db.py

```diff
@@ -25,11 +25,23 @@ class Command(BaseCommand):
 
     help = "Populate local dimail database, for dev purposes."
 
+    def add_arguments(self, parser):
+        """Add arguments to the command."""
+        parser.add_argument(
+            "--populate-from-people",
```

Comment:
How did you add this argument, locally ?
I think it would be neat to add a Make command (something like "dimail-populate-db").